### ISSUE TITLE
Rule Request: Vertical Parameter Alignment in Function Calls

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1037]
Migrated from #1033:

When refactoring, it's often very easy to mess up the vertical alignment of parameters in function calls:

```swift
validateFunction(file, kind: .function,
                    dictionary: [:])
```

We could (easily?) have a correctable rule for this.
